src/code42cli/cmds/devices.py

```diff
@@ -414,17 +415,14 @@ def bulk(state):
     pass
 
 
-@bulk.command(
-    name="deactivate",
-    help="""Deactivate all devices on the given list.
-            Takes as input a CSV with a 'guid' column.""",
-)
+@bulk.command(name="deactivate")
 @read_csv_arg(headers=["guid"])
```

Comment:
I had thought in discussion with Cecilia, we agreed that it is ok to use `computerId` for the bulk commands, since performance is the strategy.

Did we decide device GUID is fine?

Reply:
The bulk command just re-used `_deactivate_device()` from the `devices deactivate` command, which was changed to take a guid, so bulk had to be changed too. We could probably follow on with an update after release to allow either the individual or bulk to accept an `--id-type` option so the user can decide.